test/sampleData/Mcp4725.py

```diff
@@ -100,3 +107,28 @@ def set_eeprom(self, data):
             self.REGISTER_EEPROM,
             data
         )
+
+    def get_digitalout(self):
+        """
+        Only allows you to send fully on or off
+
+        """
+        # Read register data
+        # '#/registers/EEPROM' > 'EEPROM'
+        val = self.get_eeprom()
+        val = _swap_endian(val)
+        # Mask register value
+        val = val & 0b0000111111111111
+        return val
+
+    def set_digitalout(self, data):
+        """
+        Only allows you to send fully on or off
+
+        """
+        # Read current register data
+        # '#/registers/EEPROM' > 'EEPROM'
+        register_data = self.get_eeprom()
+        register_data = register_data | data
```

Comment:
I don't think we do, as the `data` input should already be an appropriate value.

Reply:
This is going to be far easier to debug once we have physical hardware to test with. I'm not 100% sure that everything is ok, but once the hardware comes in, we will immediately know if anything is wrong. I'd say it is fine as is for now to avoid holding this PR up any longer.